### PR TITLE
Uplink surplus re-balance.

### PR DESCRIPTION
**Pull request by ghost**
Lordsaladin(git)/lord saladin(skey)
[Changelogs]: 

:cl: 
tweak: Tweaks surplus chances( you should get less of  emp kits, karps,ammo,edaggers,etc and a rare chance of holoparasite kits).
/:cl:

[why]: Hello as many players know surplus crates and null crates are handy, Unfortunately you tend to get screwed over by cheap items i've changed some surplus = lines to make them rarer, and added medibeam guns to the traitor game modes so you can now have the power of the Odysseus in your hands  .

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -390,15 +392,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			organic host as a home base and source of fuel."
 	item = /obj/item/storage/box/syndie_kit/guardian
 	cost = 18
-	surplus = 0
+	surplus = 10
```
Comment (vuonojenmustaturska) [inline code/modules/uplink/uplink_items.dm]: uhh

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -390,15 +392,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	player_minimum = 25
 	restricted = TRUE
+	/datum/uplink_item/dangerous/guardian/get_discount()
+	return pick(4;0.8,2;0.65,1;0.5)
```
Comment (vuonojenmustaturska) [inline code/modules/uplink/uplink_items.dm]: What are you even trying to do here?

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (lordsaladin) [reply code/modules/uplink/uplink_items.dm]: get a discount?

**Comment by optimumtact** [comment]
maybe learn how to code first

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (vuonojenmustaturska) [reply code/modules/uplink/uplink_items.dm]: That's not how any of this works.

**Comment by Armhulen** [comment]
Good first pr, but you should really atomize changing all the chances to get items in the surplus and the medibeam being a traitor item. seems like two radically different balance changes in the same pr.

Having your name be a meme will also probably get a maintainer angry at you, so just keep that in mind.

Finally, you've made this pr off of your master. You're going to have to talk to someone who knows git to fix that because your changes in this pr will carry over to every other pr you make. I recommend oranges as he's shown he knows a whole lot about git.


oh yeah there's oranges

**Comment by lordsaladin** [comment]
Look try to code at 10:45pm.
also my name is not a meme.
it's a reference.

**Comment by lordsaladin** [comment]
@optimumtact Don't be rude.

**Comment by SpaceManiac** [comment]
Where you've written "The I'm fed up of only getting karps from my surplus/nullcrates PR." is where your name is supposed to go, not a title for the PR.

**Comment by vuonojenmustaturska** [comment]
Ok so first of all: making unlisted nontrivial balance changes is EXTREMELY frowned upon. Holoparasites cost what they cost and are available where they are because they're extremely powerful for any antag who isn't a ling. Also your discount proc was indented wrongly, byond effectively has functional indentation that you should pay attention to.

Second of all, don't make "changed some things" type log entries, err on the side of too much detail rather than too little. Players rely on these changelogs to learn about changes to the game.

**Comment by Armhulen** [comment]
Yeah, that's what we mean. I don't think oranges was trying to be too mean, just as a matter of fact the stuff in the pr that isn't number changes was kind of coding gibberish.

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (lordsaladin) [reply code/modules/uplink/uplink_items.dm]: it's a rare chance unless 10 is common.

**Comment by lordsaladin** [comment]
@optimumtact 
@vuonojenmustaturska 
@Armhulen is that better?

**Comment by TribeOfBeavers** [comment]
No, you would replace " The I'm fed up of only getting karps from my surplus/nullcrates PR." with your name/ckey/whatever

**Comment by KorPhaeron** [comment]
Please don't roleplay in your PR comments either, I just want to read what you're changing and why

**Comment by lordsaladin** [comment]
@optimumtact 
@Armhulen 
@SpaceManiac 
@vuonojenmustaturska 
@TribeOfBeavers 
@KorPhaeron 
Is that better?.

**Review by ghost — COMMENTED** [review]
is this ok?

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -390,15 +392,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			organic host as a home base and source of fuel."
 	item = /obj/item/storage/box/syndie_kit/guardian
 	cost = 18
-	surplus = 0
+	surplus = 1
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	player_minimum = 25
-	restricted = TRUE
```
Comment (DaxDupont) [inline code/modules/uplink/uplink_items.dm]: you chumbis don't remove this

Comment (DaxDupont) [reply code/modules/uplink/uplink_items.dm]: It's a restricted item for a reason.

Comment (DaxDupont) [reply code/modules/uplink/uplink_items.dm]: Which is for admin memes and VR

**Review by DaxDupont — CHANGES_REQUESTED** [review]

**Comment by epochayur** [comment]
what a rollercoaster

**Comment by lordsaladin** [comment]
@vuonojenmustaturska please re review

**Comment by lordsaladin** [comment]
atomized.

**Comment by Shadowflame909** [comment]
Oof. Good idea. Needed more practice so oranges would take them seriously. F

**Pull request closed without merging**